### DISCUSSION TITLE
Is MAS allowed to reference or include some lyrics of a song in a topic, or can that violate unfriendly copyright policies?

### DISCUSSION THREAD

**Discussion by Justformas** [Monika-After-Story/MonikaModDev#9170] [QA - tech support + general] [answered]
I couldn't really find an answer on this. There are topics with lyrics of many songs that Monika can "sing" (without the actual music or a voice), but are those all songs with friendly copyright policies? Or is that allowed for almost any song? I do have quite an example of something that might fit right in the mod, and I'll say it if someone wants, but I doubt that it's among songs with a more friendly copyright policy.

**Comment by ThePotatoGuy** [comment, marked as answer]
I think its okay in general to copy song lyrics as long as we're not making a new song.

**Reply by Justformas** [reply]
Yeah I don't mean making a new song or music, but so in general it should be ok to make a "song topic" that has Monika's dialogue using (or "singing") some of the lyrics, without any accompanying music or voice. That is, as long as such a topic is accepted.
